pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseSingleStageBrokerRequestHandler.java

```diff
@@ -485,6 +485,16 @@ protected BrokerResponse handleRequest(long requestId, String query, SqlNodeAndO
       if (realtimeTableName == null) {
         realtimeTableConfig = null;
       }
+
+      if ((realtimeTableConfig != null && _routingManager.getDisabledTables()
```

Comment:
We should allow one table being disabled when querying both offline and real-time table.